dotcms-postman/src/main/resources/postman/GraphQLTests.json

```diff
@@ -7832,6 +7832,300 @@
 				}
 			]
 		},
+		{
+			"name": "Page API - Testing 'page' field with inline fragments",
+			"item": [
+				{
+					"name": "Test Destination page with content-type-specific fields",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Destination page returns content-type-specific fields via inline fragments\", function () {",
+									"    var jsonData = pm.response.json();",
+									"    var page = jsonData.data.page;",
+									"    ",
+									"    // Verify the page field exists and is the contentlet",
+									"    pm.expect(page.page, 'page field should exist').to.exist;",
+									"    pm.expect(page.page.contentType, 'should be Destination type').to.equal('Destination');",
+									"    ",
+									"    // Test DESTINATION-SPECIFIC fields (only available via inline fragment)",
+									"    pm.expect(page.page.activities, 'activities should exist').to.exist;",
+									"    pm.expect(page.page.activities.length, 'should have activity items').to.be.greaterThan(0);",
+									"    pm.expect(page.page.image, 'image should exist').to.exist;",
+									"    pm.expect(page.page.image.name, 'image name should exist').to.exist;",
+									"    ",
+									"    // Test page-level fields are accessible at root",
+									"    pm.expect(page.tags, 'page tags should exist').to.equal('diving');",
+									"    pm.expect(page.shortDescription, 'page shortDescription should exist').to.exist;",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "username",
+									"value": "admin@dotcms.com",
+									"type": "string"
+								},
+								{
+									"key": "password",
+									"value": "admin",
+									"type": "string"
```

Comment:
This should not be included here. I mean it is valid and it will work, but our postman are organized in such a way that they can use the json web token that is defined in auth on the parent element of the collection. 
If you initialize that element as a collection variable with a bearer token will be set. 

Authentication type must be Bearer token to stick with this pattern